core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2278,6 +2278,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN =
+      new Builder(Name.USER_BLOCK_MASTER_CLIENT_POOL_SIZE_MIN)
+          .setDefaultValue(0)
+          .setDescription("The minimum number of block master clients cached in the block master "
+              + "client pool. For LLAP, this should be set to zero.")
```

Comment:
why is this recommendation only for llap?

Reply:
for other frameworks, the min can be set to non-zero to always keep a connection alive (may be good for performance). for llap, it has to be zero has the long running daemon does not release all clients

Reply:
But what about other long-running processes? For example, all of these have long-running jvms: presto workers, hive metastore, hiveserver2, etc

Reply:
Potentially yes (if context is not released), but I'm not sure